authenticate-with-oauth/README.md

```diff
@@ -12,71 +12,57 @@ Your app may need to access items that are only shared with authorized users. Fo
 
 When you run the sample, the app will load a web map which contains premium content. You will be challenged for an ArcGIS Online login to view the private layers. Enter a user name and password for an ArcGIS Online named user account (such as your ArcGIS for Developers account). If you authenticate successfully, the traffic layer will display, otherwise the map will contain only the public basemap layer.
 
-## Use of Model View ViewModel (MVVM)
-
-This sample takes advantage of Android's `ViewModel` to encapsulate launching the OAuth user sign in prompt, establishing the activity result contract, and verifying the OAuth user's credentials.
-
-1. An Activity which launches a custom tab intent and sets its own result to the redirect URI of the custom tab's result
-2. A ViewModel which launches the above activity when prompted, and when it receives the result, passes it through to the `OAuthUserSignIn` object
-3. An `ArcGISAuthenticationChallengeHandler` which prompts the ViewModel to start the sign in process with a URL if it is valid.
-
-![Image of a flowchart explaining the OAuth sample](oauth-sample-flowchart.png)
-
 ## How it works
 
-1. Create an `OAuthConfiguration` specifying the portal URL, client ID, and redirect URI.
-2. Create an `OAuthUserSignInActivity` that will launch the sign in page in a Custom Tab and receive & process the redirect intent when completing the Custom Tab prompt.
-3. Set up `ViewModel` responsible for launching an OAuth user sign in prompt and retrieving the authorized redirect URI
-4. Crete an `ArcGISAuthenticationChallengeHandler` that checks if the `ArcGISAuthenticationChallenge.requestUrl` matches the `OAuthUserConfiguration's`  with `canBeUserdForUrl()`. If the challenge matches, then create a `OAuthUserCredential` by invoking the `OAuthUserSignInViewModel.promptForOAuthUserSignIn`
-5. Set the `AuthenticationManager`'s `arcGISAuthenticationChallengeHandler` to the `ArcGISAuthenticationChallengeHandler` created above.
-6. Load a map with premium content requiring authentication to automatically invoke the `ArcGISAuthenticationHandler`.
-
-#### Setting up the Manifest.xml:
-
-1. Set the launch mode for the `OAuthUserSignInActivity` to route it's intent instance through a call to its `onNewIntent()` method, rather than creating a new instance of the activity. To find out more on setting the launch configuration of an activity, visit the [Android docs](https://developer.android.com/guide/topics/manifest/activity-element)
+1. Create an `AuthenticatorState` and set the `OAuthConfiguration` with the portal URL, client ID, and redirect URI.
+2. Load a `PortalItem(...)` with connection type `Portal.Connection.Authenticated` which will issue an authentication challenge.
+3. Create a `DialogAuthenticator` composable passing it the `AuthenticatorState` object. Because the `AuthenticatorState` object has an `OAuthConfiguration` set, the `DialogAuthenticator` will prompt for OAuth credentials.
```

Comment:
Instead of "Create" I suggest to use the term "Call" here, since we are calling a composable function. Also would mention "toolkit":
```suggestion
3. Call the toolkit's `DialogAuthenticator` composable passing it the `AuthenticatorState` object. Because the `AuthenticatorState` object has an `OAuthConfiguration` set, the `DialogAuthenticator` will prompt for OAuth credentials when the associated `Portal` is loaded.
```